docs/typescript.md

```diff
@@ -99,12 +51,14 @@ yarn add -D typescript @types/jest @types/react @types/react-native @types/react
 }
 ```
 
-3. Create a `jest.config.js` file to configure Jest to use TypeScript
```

Comment:
Jest, at least in the current version, supports ts, tsx out of the box, and we don't need to add a new config anymore.